### PR TITLE
Using python3 print format to python2 %

### PR DESCRIPTION
**Pull request by michaelbadcrumble**
Doing some cleaning in the install command, simply switch Python2 % for Python3 format.

### CODE REVIEW

File: mesonbuild/minstall.py [hunk 1]
```diff
@@ -181,6 +183,9 @@ def restore_selinux_contexts():
 
 
 def get_destdir_path(d, path):
+    '''
+    here we dist directory path
```
Comment (nirbheek) [inline mesonbuild/minstall.py]: What does this comment mean?

File: mesonbuild/minstall.py [hunk 2]
```diff
@@ -76,7 +76,9 @@ def __exit__(self, exception_type, value, traceback):
             append_to_log(self.lf, d)
 
 def is_executable(path, follow_symlinks=False):
-    '''Checks whether any of the "x" bits are set in the source file mode.'''
+    '''
+    Here checks whether any of the "x" bits are set in the source file mode.
```
Comment (QuLogic) [inline mesonbuild/minstall.py]: Do not add "Here" to the front; in fact it should be imperative tense, see [PEP257](https://www.python.org/dev/peps/pep-0257/).

File: mesonbuild/minstall.py (hunk 1 above)
Comment (michaelbadcrumble) [reply mesonbuild/minstall.py]: This comment was a mistake and is removed.

File: mesonbuild/minstall.py (hunk 2 above)
Comment (michaelbadcrumble) [reply mesonbuild/minstall.py]: Fixed.

Comment (michaelbadcrumble) [reply mesonbuild/minstall.py]: Thanks for the reference.

**Comment by nirbheek** [comment]
Did you close this PR on purpose?

**Comment by michaelbadcrumble** [comment]
I wasn’t particularly happy with that PR.  So I created #6725 since I originally wanted to cherry-pick Python2 syntax.  Also, note-to-self no PR should be created late at night or without having a cup coffee first.

**Pull request closed without merging**